test/mock/galera_test.sh

```
@@ -0,0 +1,96 @@
+
+run dbdeployer available
+for version in ${versions[*]}
+do
+    version_name=$(echo $version | tr '.' '_')
+    run dbdeployer deploy replication $version --topology=galera
+
+    test_completeness $version galera_msb_ multiple
+    check_sst_method galera_msb_$version_name rsync
+    ok_dir_exists "$SANDBOX_HOME/galera_msb_$version_name"
+
+    if dbdeployer deploy replication $version --topology=pxc >/tmp/galera-pxc-reject.log 2>&1
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)